cmd/snap-confine/snap-confine.c

```diff
@@ -88,6 +89,26 @@ int main(int argc, char **argv)
 		die("need to run as root or suid");
 	}
 #endif
+
+	char *snap_context __attribute__ ((cleanup(sc_cleanup_string))) = NULL;
+	const char *snap_name = getenv("SNAP_NAME");
+	if (snap_name != NULL) {
```

Comment:
`SNAP_NAME` cannot be NULL so I guess you can de-indent the whole part a little. If SNAP_NAME is NULL we should die as well.